CMakeLists.txt

```
@@ -245,21 +258,28 @@ endif()
 # checked but some systems define them even when they do not support it
 # (ref. bugreport:1003).
 #
-message( STATUS "Check for monotonic clock" )
-find_library( RT_LIBRARY rt )# (optional, rt on Debian)
-mark_as_advanced( RT_LIBRARY )
-set( CMAKE_REQUIRED_LIBRARIES ${GLOBAL_LIBRARIES} ${RT_LIBRARY} )
-file( READ "${CMAKE_SOURCE_DIR}/3rdparty/cmake/tests/HAVE_MONOTONIC_CLOCK.c" _SOURCE )
-CHECK_C_SOURCE_RUNS( "${_SOURCE}" HAVE_MONOTONIC_CLOCK )
-if( HAVE_MONOTONIC_CLOCK )
-	message( STATUS "Check for monotonic clock - yes" )
-	set_property( CACHE GLOBAL_LIBRARIES    PROPERTY VALUE ${GLOBAL_LIBRARIES} ${RT_LIBRARY} )
-	set_property( CACHE GLOBAL_DEFINITIONS  PROPERTY VALUE "${GLOBAL_DEFINITIONS} -DHAVE_MONOTONIC_CLOCK" )
+if(NOT APPLE)
+	message( STATUS "Check for monotonic clock" )
+	find_library( RT_LIBRARY rt )# (optional, rt on Debian)
+	mark_as_advanced( RT_LIBRARY )
+	set( CMAKE_REQUIRED_LIBRARIES ${GLOBAL_LIBRARIES} ${RT_LIBRARY} )
+	file( READ "${CMAKE_SOURCE_DIR}/3rdparty/cmake/tests/HAVE_MONOTONIC_CLOCK.c" _SOURCE )
+	CHECK_C_SOURCE_RUNS( "${_SOURCE}" HAVE_MONOTONIC_CLOCK )
+	if( HAVE_MONOTONIC_CLOCK )
+		message( STATUS "Check for monotonic clock - yes" )
+		set_property( CACHE GLOBAL_LIBRARIES    PROPERTY VALUE ${GLOBAL_LIBRARIES} ${RT_LIBRARY} )
+		set_property( CACHE GLOBAL_DEFINITIONS  PROPERTY VALUE "${GLOBAL_DEFINITIONS} -DHAVE_MONOTONIC_CLOCK" )
+	else()
+		message( STATUS "Check for monotonic clock - no" )
+	endif()
 else()
-	message( STATUS "Check for monotonic clock - no" )
+	message( STA	TUS "Check for monotonic clock - no" )
```

```suggestion
	message( STATUS "Check for monotonic clock - yes" )
```
Based on your comment below?